source/fundamentals/typescript.txt

```diff
@@ -44,8 +44,8 @@ For more information on object types, see the
 Extend Document
 ~~~~~~~~~~~~~~~
 
-The following classes accept any type that extends the ``Document``
-interface:
+The following classes accept any :ref:`non-recursive <node-driver-recursive-type>`
```

Comment:
How about "The following classes accept all non-recursive types that extend ..." ?
This could avoid confusion with the "any" typescript type and "all" is a stronger specifier.

Reply:
Does it make sense to link non-recursive type to the examples that show a recursive type? Instead of this link, how about adding a sentence here that introduces the recursive type example for anyone not familiar?